### PR TITLE
Collective navigation v2: new section header

### PR DESCRIPTION
**Pull request by sbinlondon**
Screenshot 2020-11-04 at 16 49 32

### CODE REVIEW

**Comment by sbinlondon** [comment]
Relates https://github.com/opencollective/opencollective/issues/3638

File: components/collective-page/SectionHeader.js
Comment (znarf) [inline components/collective-page/SectionHeader.js:20]: Let's avoid versioning ("v2") in folder names.

Also "navbar" doesn't seem the right name because these are section headers.

**Review by znarf — APPROVED** [review]
Looks good, would be great to have second review from @Betree .

**Review by Betree — COMMENTED** [review]
Looking good :+1:

**Pull request merged**
